src/greyVest/Popover.js

```diff
@@ -1,32 +1,44 @@
 import React from 'react'
 import _ from 'lodash/fp'
-import OutsideClickHandler from 'react-outside-click-handler'
-import { observer } from 'mobx-react'
+import F from 'futil'
+import Popup from 'reactjs-popup'
 import { openBinding } from './utils'
 import { expandProp } from '../utils/react'
 
-// Simple popover
-let Popover = ({ isOpen, onClose, children, style, contained }) =>
-  isOpen && (
-    <OutsideClickHandler onOutsideClick={_.debounce(0, onClose)}>
-      <div style={{ position: contained ? '' : 'relative' }}>
-        <div
-          className="popover"
-          style={{
-            position: 'absolute',
-            Index: 100,
-            fontWeight: 'normal',
-            textAlign: 'left',
-            background: 'white',
-            border: '1px solid #ebebeb',
-            zIndex: 20,
-            ...style,
-          }}
-        >
-          {children}
-        </div>
-      </div>
-    </OutsideClickHandler>
-  )
+let Popover = ({
+  // eslint-disable-next-line no-unused-vars
+  open, // lens
```

Comment:
if you're using expandProp with `openBinding`, the idea is that you should be coding against `isOpen`/`onClose`. `openBinding` is about adding lens support to a standard API - you should pick one as the canonical API you code against and then at most adapt to the other. In general, we've tried to move away from lenses a bit as most of the team find the abstraction to be more overhead than it's worth which is actually where that pattern of expanding with a binding came from - we add lens support on top, but most of the code doesn't deal with them

Reply:
Changed to using `explodeProp` util I just created. It omits the `open` key